src/aubuf/aubuf.c

```diff
@@ -357,6 +416,9 @@ void aubuf_read_auframe(struct aubuf *ab, struct auframe *af)
 		read_auframe(ab, af);
 	}
 
+	if (ab->fade_c < FADE_MAX)
+		aubuf_fade(ab, af);
```

Comment:
I`m unsure if we should handle this in aubuf. Maybe its better to add a register callback function for over-/underrun detection and a application or aufilter can decide how to handle this case.

Reply:
I stored the fade in for a later PR. For now we need only the ajb reset.